### DISCUSSION TITLE
2.3.0 releas

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#2064] [General]
@brockfanning @LucyGwilliamAdmin 

Hello, when are you planning to release 2.3.0?

I have some data with comments on the observation level, so I would upgrade to 2.3.0

**Comment by sarah-beasley** [comment]
Hi Christoph, we are working on 2.3.0 to be released in the next month or so, so hopefully this won't be too much longer for you to wait!

**Reply by christoph-LSN** [reply]
@sarah-beasley ok, that is not too long
